gsl.yaml

```diff
@@ -2,7 +2,7 @@
 package:
   name: gsl
   version: "2.8"
-  epoch: 0
```

Comment:
fwiw we don't need to bump the epoch of gsl itself, just the downstream dependencies, which you have done correctly with `dieharder` and `lsb-release-minimal`